go.mod

```diff
@@ -5,19 +5,24 @@ go 1.16
 require (
 	github.com/charmbracelet/glamour v0.3.0
 	github.com/gorilla/mux v1.8.0
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.6.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
```

Comment:
Did we remove new-relic in favor of Prometheus?

Reply:
This is a dependency from the added example. These are the once given as example in https://github.com/grpc-ecosystem/go-grpc-middleware package, so just used the same here.